pkg/deploy/deploy.go

```diff
@@ -134,7 +137,7 @@ func patchLoadChartfile(chartName string) {
 			Version: "0.1.0",
 			Engine:  helm.WerfTemplateEngineName,
 		}
-
+		mu.Unlock()
```

Comment:
It is better to do `defer mu.Unlock()` right after `mu.Lock()` so that failed `LoadChartfileFunc` function (err != nil) in some thread will not deadlock loading process for other threads.

Reply:
Good point. I fixed this.